eng/Packages.Data.props

```diff
@@ -115,6 +115,7 @@
     <PackageReference Update="CloudNative.CloudEvents" Version="2.0.0" />
     <PackageReference Update="CloudNative.CloudEvents.SystemTextJson" Version="2.0.0" />
     <PackageReference Update="Microsoft.AspNetCore.SignalR.Protocols.NewtonsoftJson" Version="3.0.0" Condition="'$(TargetFramework)' == 'netcoreapp3.1'" />
+    <PackageReference Update="Microsoft.AspNetCore.SignalR.Protocols.NewtonsoftJson" Version="5.0.1" Condition="'$(TargetFramework)' == 'net5.0'" />
```

Comment:
Why do we need different package versions? Can we not use the later version for both of those target frameworks?

Reply:
The dependency `Microsoft.Azure.SignalR.Management` uses different versions, so to mitigate the issue before the dependency updates, we have to use different versions. We plan to remove the dependency for `Microsoft.AspNetCore.SignalR.Protocols.NewtonsoftJson` in the `Microsoft.Azure.SignalR.Management` later and use `Azure.Core.Newtonsoft` instead.